src/scaffoldmaker/meshtypes/meshtype_3d_brainstem.py

```diff
@@ -640,7 +576,7 @@ def defineFaceAnnotations(cls, region, options, annotationGroups):
             subGroup = AnnotationGroup(region, get_brainstem_term(subregion))
             issub = subGroup.getFieldElementGroup(mesh2d)
             is_subface = fm.createFieldOr(fm.createFieldAnd(issub, is_exterior_face_xi1), fm.createFieldAnd(issub, is_exterior_face_xi3))
-            subFaceGroup = findOrCreateAnnotationGroupForTerm(annotationGroups, region, (subregion + '_exterior', None))
+            subFaceGroup = findOrCreateAnnotationGroupForTerm(annotationGroups, region, get_brainstem_term(subregion))
             subFaceGroup.getMeshGroup(mesh2d).addElementsConditional(is_subface)
```

Comment:
Rename subregion to groupName, namelist to groupNames. A subregion would be a different object (a zinc Region)
This doesn't look like it should work, maybe change to:
```
exteriorGroupName = groupName + " exterior"
subFaceGroup = findOrCreateAnnotationGroupForTerm(annotationGroups, region, get_brainstem_term(exteriorGroupName))
```